spec/integration_spec.rb

```diff
@@ -727,17 +748,17 @@
 describe 'Will_paginate' do
   before(:all) do
     require 'will_paginate'
-    MeiliSearch::Rails.configuration = {
-      meilisearch_url: ENV.fetch('MEILISEARCH_HOST', 'http://127.0.0.1:7700'),
-      meilisearch_api_key: ENV.fetch('MEILISEARCH_API_KEY', 'masterKey'), pagination_backend: :will_paginate
-    }
+    MeiliSearch::Rails.configuration[:pagination_backend] = :will_paginate
     Movie.clear_index!(true)
 
     10.times { Movie.create(title: Faker::Movie.title) }
 
     Movie.reindex!(MeiliSearch::Rails::IndexSettings::DEFAULT_BATCH_SIZE, true)
+    sleep 5
   end
 
+  after(:all) { MeiliSearch::Rails.configuration[:pagination_backend] = nil }
```

Comment:
I don't believe the `after` hook should be used at all, I really like this article from cypress that we could also adopt here 

https://docs.cypress.io/guides/references/best-practices#Using-after-Or-afterEach-Hooks

Reply:
So I would like to suggest to you remove this later in a test suite refactor!

Reply:
Added to #300